### PR TITLE
Added generator script for life-like data.

### LINKED ISSUE #107: Fixture with test data that looks real
Use Saif's 8.05x course

### PR DESCRIPTION
**Pull request by justinabrahms**
#### What are the relevant tickets?

Fixes #107 
#### What's this PR do?

Adds a fixture based on the quantum mechanics course [here](https://github.mit.edu/edX-Archives/edX-Archive-8.MechCx-1T2015)
#### Where should the reviewer start?

`gen_realish_data.py`
#### How should this be manually tested?

Delete all courses/modules in your system. Run the script. You'll have your very own life-like course!

### CODE REVIEW

**Comment by Ferdi** [comment]
:-)

File: courses/management/commands/gen_realish_data.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                m.delete()
+
+        except Course.DoesNotExist:
+            c = CourseFactory.create(
+                edx_instance = bi,
+                title=course_title,
+                author_name='David E. Prichard',
+                description='Mechanics ReView presents a college-level introductory mechanics class using a strategic problem-solving approach.',
+                overview='For more information, see <a href="#">edx.org</a>',
+                image_url='https://www.edx.org/sites/default/files/styles/course_video_banner/public/course/image/featured-card/8.05.3x-video_still-318x210.jpg?itok=DJOP5xRS',
```
Comment (pdpinch) [inline courses/management/commands/gen_realish_data.py]: This image goes with this course:

https://www.edx.org/sites/default/files/styles/course_video_banner/public/course/image/featured-card/8.mechcx-378x225.jpg?itok=lDOLQ61K

File: courses/management/commands/gen_realish_data.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            )
+
+        module_names = """Beta Testers and Community TAs
```
Comment (pdpinch) [inline courses/management/commands/gen_realish_data.py]: `Beta Testers and Community TAs` is a hidden chapter. We should omit it.

**Comment by justinabrahms** [comment]
Updated to address your comments, Peter. Added tests for the management command.

File: courses/management/commands/gen_realish_data_test.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+"test realish data gen script"
```
Comment (noisecapella) [inline courses/management/commands/gen_realish_data_test.py]: Can you change all the docstrings in this file to use triple quotes? From https://www.python.org/dev/peps/pep-0257/: `For consistency, always use """triple double quotes""" around docstrings`

**Comment by noisecapella** [comment]
I have just the style change to comment on so I'll just take over this and merge it

**Comment by noisecapella** [comment]
:+1:

**Pull request merged**